docs/pages/docs/guides/custom-admin-ui-pages.mdx

```diff
@@ -14,38 +16,163 @@ Much like with Next.js, all files in this directory will be added as routes to t
+        </NavigationContainer>
+    )
+}
+```
+
+!> Under the hood Keystone's Admin UI is powered by Next.js, so the route to our custom page is the filename of our custom page component, in this case its `/custom-page`
+
+With all that in place, our custom Admin UI page is now navigable from the Admin UI Navigation component, and we can access it from other pages in the Admin UI.
+![completed custom Admin UI page](/assets/guides/custom-admin-ui-pages/custom-page-completed.png)
+
+!> This concludes the main part of the Custom Admin UI Pages guide
+
+## Styling
+
+Of course there are other styling considerations when adding a custom page to the Admin UI that go beyond making it _look_ and _feel_ like an Admin UI page.
+For this, we recommend using the `jsx` runtime export from the `@keystone-ui/core` package, as this will ensure that the version of [emotion](https://emotion.sh/docs/introduction) you're using conforms with the version of emotion used internally used by Keystone.
+
+The snippet below uses the emotion `jsx` runtime exported from `@keystone-ui/core` to help add some basic allignment and layout styling to the contents of our Admin UI custom page.
+
+```tsx
+// admin/pages/custom-page.tsx
 /** @jsxRuntime classic */
 /** @jsx jsx */
+
 import { jsx } from '@keystone-ui/core';
-export default function () {
+import { PageContainer } from '@keystone-next/keystone/admin-ui/components';
+import { Heading } from '@keystone-ui/core';
+
+export default function CustomPage () {
     return (
-        <>
+        <PageContainer
+            header={(
+                <Heading type="h3">
+                    Custom Page
+                </Heading>
+            )}>
             <h1 css={{
-                fontSize: '3rem'
-            }}>This is a custom Admin UI Page</h1>
-            <p>It can be accessed via the route '/MyCustomPage'</p>
-        </>
+                width: '100%',
+                textAlign: 'center',
+            }}>
+                This is a custom Admin UI Page
+            </h1>
+            <p css={{ textAlign: 'center' }}>
+                It can be accessed via the route <a href="/custom-page">/custom-page</a>
+            </p>
+        </PageContainer>
     )
 }
 ```
 
-Of course this is purely a recommendation, if you would prefer to roll your own css-in-js solution in with your custom component please feel free to! Although this may require additional configuration outside of the scope of this guide.
-
-x> **Not all Next.js exports are available:** Keystone **only** supports the page component as a default export in the pages directory. This means that unlike with Next, auxillary exports such as `getStaticProps` and `getServerProps` are not supported.
+Of course this is purely a recommendation, if you would prefer to use another css-in-js or css solution for your custom component please feel free to! Although this may require additional configuration currently outside of the scope of this guide.
```

Comment:
```suggestion
Using `emotion` for styling is purely a recommendation, if you would prefer to use another css-in-js or css solution for your custom component please feel free to. This may require additional configuration currently outside of the scope of this guide.
```